modules/passkey/CHANGELOG.md

```diff
@@ -9,12 +9,15 @@ Solidity compiler: [0.8.26](https://github.com/ethereum/solidity/releases/tag/v0
 Solidity optimizer: enabled with 10.000.000 runs (via IR for all contracts except `FCLP256Verifier`)
 EVM target: Paris
 
-## Expected addresses
+## Expected addresses with [Safe Singleton Factory](https://github.com/safe-global/safe-singleton-factory)
 
-- `SafeWebAuthnSignerFactory` at `TBD`
-- `SafeWebAuthnSharedSigner` at `TBD`
-- `DaimoP256Verifier` at `0xc2b78104907F722DABAc4C69f826a522B2754De4`
-- `FCLP256Verifier` at `TBD`
+- `SafeWebAuthnSignerFactory` at `0x1d31F259eE307358a26dFb23EB365939E8641195`
+- `SafeWebAuthnSharedSigner` at `0x94a4F6affBd8975951142c3999aEAB7ecee555c2`
+- `FCLP256Verifier` at `0xA86e0054C51E4894D88762a017ECc5E5235f5DBA`
+
+### Official Deployment Address from 3rd party
+
+- `DaimoP256Verifier` at `0xc2b78104907F722DABAc4C69f826a522B2754De4` ([Source](https://p256.eth.limo/))
```

Comment:
Not for this PR, but can we create an issue to deploy the `DaimoP256Verifier` correctly on chains where it isn't supported?

I don't know how easy it is to make deployment scripts that work with two different CREATE2 deployers, but it _should_ be possible.

Also, we are unnecessarily deploying the `DaimoP256Verifier` contract with the Safe Singleton proxy with the current deployment scripts, so those need to be adjusted to not deploy this contract (when not on a test network). This should also be reflected in the issue (I don't want to pull in additional changes to this PR to not slow down the review process even further).